src/Main.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+
+        Integer[]numeric = {1,2,3,4,5,6,7,8,9,0};
+        String[]alphabetic = {"een","twee","drie","vier", "vijf","zes","zeven","acht","negen","nul"};
+        Translator translator = new Translator(alphabetic,numeric);
+        String answer = translator.translate(3);
+        System.out.println(answer);
+
+        boolean play = true;
+        String ongeldig = "ongeldige invoer";
+        Scanner scanner = new Scanner(System.in);
+        while (play) {
+            System.out.println("Type 'x' om te stoppen \nType 'v' om te vertalen");
+            String input = scanner.nextLine();
+                    if (input.equals("x")) {
+                        play = false;
+                    } else if (input.equals("v" )) {
+                        System.out.println("Type een cijfer in van 0 t/m 9");
+                        int number = scanner.nextInt();
+                        scanner.nextLine();
+                        if (number < 10) {
+                            String result = translator.translate(number);
+                            System.out.println("De vertaling van " + number + " is " + result);
+                        } else System.out.println(ongeldig);
+                    } else System.out.println(ongeldig);
+
+}
```

Comment:
Moeilijk te zien welke sluitende accolades waar behoren. Probeer daar duidelijk in te zijn. Want bv bij deze lijkt het net alsof jouw main method 2 (samen met sluitende accolade van `line 34`) sluitende accolades heeft.

Een tip voor verduidelijking in je code: //met comments kun je veel verduidelijken voor jezelf (en voor een ander).

Reply:
Ook een fijne tip, ik heb dit nu gedaan in de huiswerkopdracht van deze week, hopelijk worden de accolades ook duidelijker zichtbaar qua open en sluiten.